### ISSUE TITLE
Mount Uploader in a non activerecord inherited class

### ISSUE THREAD

**Issue by norcali** [carrierwaveuploader/carrierwave#1497]
So, I have a shopping cart class I keep saved in session until the purchase is completed and I need to be able to upload a file into the cart (don't ask why, long story) once the purchase is finished, I dump all that information into a class that is saved in the DB.

I've used Carrierwave quite a lot and I didn't have any issues so far, but when I tried to mount an uploader into it I got 

`
undefined method `mount_uploader' for Cart:Class
`

The question is, is it possible to mount an uploader into a class that doesn't inherit ActiveRecord:Base or am I having another issue altogether? I haven't been able to make it work so I don't want to waste more time if that's the issue.

**Comment by dalizard** [comment]
@bensie This is a very specific question, and the use case is very well explained. What better place to answer that here? I definitely don't think that Stack Overflow is a better place for such questions.

**Comment by bensie** [comment]
@dalizard Stack Overflow is the right place for this. We (maintainers) are not available for support questions, we just don't have the time. Questions can go unanswered for months, where they typically get an answer within a few hours on SO.
